packages/@webex/plugin-meetings/src/meeting-info/util.ts

```diff
@@ -33,282 +33,290 @@ import {
   UUID_REG,
 } from '../constants';
 
-const MeetingInfoUtil: any = {};
-
-MeetingInfoUtil.extractDestination = (destination, type) => {
-  let dest = destination;
-
-  if (type === _LOCUS_ID_) {
-    if (!(destination && destination.url)) {
-      throw new ParameterError('You cannot create a meeting by locus without a locus.url defined');
+/**
```

Comment:
The refactor of util.ts is required as without this refactor an empty declaration file was being created.

For the refactor 
1. Create a class MeetingInfoUtil
2. All the method inside the class are static. No src code changes are made.

Reply:
Thanks for this comment. I like it :-)